pygments/lexers/installers.py

```diff
@@ -281,9 +281,9 @@ class DebianControlLexer(RegexLexer):
             (r'^(Maintainer|Uploaders)(:\s*)', bygroups(Keyword, Text),
              'maintainer'),
             (r'^((?:Build-|Pre-)?Depends(?:-Indep|-Arch)?)(:\s*)',
-             bygroups(Keyword, Text), 'depends'),
-            (r'^(Recommends|Suggests|Enhances)(:\s*)', bygroups(Keyword, Text),
-             'depends'),
+             bygroups(Keyword, Text), 'package_list'),
+            (r'^(Recommends|Suggests|Enhances|Breaks|Replaces|Provides)(:\s*)', bygroups(Keyword, Text),
```

Comment:
Terve!

Hmm, while you're at it, let's add the [`Conflicts`](https://www.debian.org/doc/debian-policy/ch-relationships.html#conflicting-binary-packages-conflicts) field, too?

```suggestion
            (r'^(Recommends|Suggests|Enhances|Breaks|Replaces|Provides|Conflicts)(:\s*)', bygroups(Keyword, Text),
```

Reply:
done, thanks, not sure how I missed that.